drivers/filesystems/ntfs/fcb.c

```diff
@@ -143,6 +143,12 @@ NtfsFCBIsCompressed(PNTFS_FCB Fcb)
     return ((Fcb->Entry.FileAttributes & NTFS_FILE_TYPE_COMPRESSED) == NTFS_FILE_TYPE_COMPRESSED);
 }
 
+BOOLEAN
+NtfsFCBIsEncrypted(PNTFS_FCB Fcb)
+{
+    return ((Fcb->Entry.FileAttributes & NTFS_FILE_TYPE_ENCRYPTED) == NTFS_FILE_TYPE_ENCRYPTED);
```

Comment:
The `==` part seems overkill to check one bit only, even though some existing code does it that way.

Reply:
It's consistent with the existing code, so that should be fine